tcms/testplans/templates/testplans/get.html

```diff
@@ -76,5 +76,57 @@ <h2 class="card-pf-title">
             </div>
         </div>
     </div>
+        <div class="row row-cards-pf">
+            <div class="col-xs-12 col-sm-12 col-md-12">
+                <div class="card-pf card-pf-accented">
+                    <div id="confirmed-testcases" class="list-group list-view-pf list-view-pf-view">
+                    </div>
+                </div>
+            </div>
+        </div>
 </div>
+
+<template id="no_test_cases">
+    <div class="list-group-item">
+        <div class="list-group-item-header">
+            <div class="list-view-pf-main-info">{% trans 'No testcases' %}</div>
```

Comment:
Use `No records found` b/c it is used in search tables and will make translations more consistent.